File: packages/jsii-pacmak/test/expected.jsii-calc/java/src/main/java/software/amazon/jsii/tests/calculator/StabilityTest.java
package software.amazon.jsii.tests.calculator;

/**
 * This enum is there to test various stability levels are correctly emitted.
 */
@javax.annotation.Generated(value = "jsii-pacmak")
@software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Stable)
@software.amazon.jsii.Jsii(module = software.amazon.jsii.tests.calculator.$Module.class, fqn = "jsii-calc.StabilityTest")
public enum StabilityTest {
    /**
     * @deprecated yeah this one's no good
     */
    @Deprecated
    @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
    DeprecatedMember,
    /**
     * EXPERIMENTAL
     */
    @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Experimental)
    ExperimentalMember,
    /**
     */
    @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Stable)
    StableMember,
}

File: packages/jsii-pacmak/test/expected.jsii-calc/dotnet/Amazon.JSII.Tests.CalculatorPackageId/Amazon/JSII/Tests/CalculatorNamespace/StabilityTest.cs
using Amazon.JSII.Runtime.Deputy;

namespace Amazon.JSII.Tests.CalculatorNamespace
{
    /// <summary>This enum is there to test various stability levels are correctly emitted.</summary>
    /// <remarks>stability: Stable</remarks>
    [JsiiEnum(nativeType: typeof(StabilityTest), fullyQualifiedName: "jsii-calc.StabilityTest")]
    public enum StabilityTest
    {
        /// <remarks>
        /// deprecated: yeah this one's no good
        /// stability: Deprecated
        /// </remarks>
        [JsiiEnumMember(name: "DeprecatedMember")]
        DeprecatedMember,
        /// <remarks>stability: Experimental</remarks>
        [JsiiEnumMember(name: "ExperimentalMember")]
        ExperimentalMember,
        /// <remarks>stability: Stable</remarks>
        [JsiiEnumMember(name: "StableMember")]
        StableMember
    }
}